package.json

```diff
@@ -111,7 +111,7 @@
   "optionalDependencies": {
     "@discordjs/uws": "^11.149.1",
     "canvas": "^2.8.0",
-    "erlpack": "github:discordapp/erlpack",
+    "erlpack": "github:almeidx/erlpack#f0c535f73817fd914806d6ca26a7730c14e0fb7c",
```

Comment:
You said it was renamed to discord, but why its using from your account @almeidx ?

Reply:
That was just a side note. Erlpack doesn't support node 16 so I made a fork of it

Reply:
oh, got it. If the origin erlpack start to support node 16, I think we should require the original package then, since having to depend on someone else's package is kinda worrying because of relying on updates